tests/unit_test/data_processor/customer_order_processor_test.py

```
@@ -526,3 +509,358 @@ def test_filter_orders_wrong_persona_type_no_results(self):
+                    order_details={"amount": 0},
+                    callback_url="https://cb.url",
+                )
+
+    def test_amount_converted_to_paise(self):
+        mock_resp = MagicMock()
+        mock_resp.ok = True
+        mock_resp.json.return_value = {"id": "pay_paise", "short_url": "https://rzp.io/p"}
+        with patch("kairon.shared.data.customer_order_processor.http_requests.post") as mock_post:
+            mock_post.return_value = mock_resp
+            CustomerOrderProcessor._create_razorpay_payment_link(
+                api_key="k", api_secret="s", order_id="o3",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "api_secret"

(S106)